python/nx-cugraph/nx_cugraph/algorithms/bipartite/basic.py

```diff
@@ -0,0 +1,31 @@
+# Copyright (c) 2024, NVIDIA CORPORATION.
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+import cupy as cp
+
+from nx_cugraph.algorithms.cluster import _triangles
+from nx_cugraph.convert import _to_graph
+from nx_cugraph.utils import networkx_algorithm
+
+__all__ = [
+    "is_bipartite",
+]
+
+
+@networkx_algorithm(plc="triangle_count", version_added="24.02")
```

Comment:
would it help to rename `plc` to `_plc` to signal that _this parameter doesn't matter if you don't know what it's for_?

Reply:
We may want to use `version_added` to update the docstring, but I haven't fully baked that yet.

We could also make `plc` a _code comment_, which is a less useful, but also potentially less confusing.

Reply:
yeah, a leading underscore could help emphasize these are "internal" params.  I was mainly bringing it up for awareness, and this can (and probably should, unless you'd actually prefer to address it here) be done in a separate PR.

Reply:
ooh, so we have a reason to merge all open nx-cugraph PRs ;)